testing/test_runner.py

```diff
@@ -762,6 +765,27 @@ def test_importorskip_imports_last_module_part() -> None:
     assert os.path == ospath
 
 
+def test_importorskip_importError_Exception() -> None:
+    ## Mocking the import function to raise a importError
+    realimport = builtins.__import__
```

Comment:
Please use [monkeypatch](https://docs.pytest.org/en/stable/how-to/monkeypatch.html) instead of hand-rolling this.